README.md

```diff
@@ -13,15 +13,17 @@ To install the `microsoft-graph-core` library with Composer, either run `compose
 ```
 ## Get started with Microsoft Graph
 
-### Register your application
+### 1. Register your application
 
-Register your application to use the Microsoft Graph API using [Microsoft Azure Active Directory](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) in your tenant's Active Directory to support work or school users for your tenant, or multiple tenants.
+Register your application to use the Microsoft Graph API by following the steps at [Register an an application with the Microsoft Identity platform](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
```

Comment:
Consider making this an aka.ms link so that we can reuse and manage the link in a single place.

Reply:
Addressed in latest changes